src/Components/Web.JS/src/Platform/WebAssemblyProgressReporter.ts

```diff
@@ -0,0 +1,28 @@
+import { Blazor } from "../GlobalExports";
+
+export class WebAssemblyProgressReporter {
+    /**
+    * Modifies CSS of the loading ring to reflect linear progress based on the number of resources.
+    * @param resourcesTotal The total number of resources retrieved from MonoPlatform.ts
+    * @param resourcesLoaded The current number of resources loaded retrieved from WebAssemblyResourceLoader.ts
```

Comment:
Can we reference these classes using TSDoc?

https://api-extractor.com/pages/tsdoc/declaration_references/